human_aware_rl/rllib/utils.py

```diff
@@ -1,18 +1,36 @@
 from overcooked_ai_py.agents.benchmarking import AgentEvaluator
+from overcooked_ai_py.mdp.layout_generator import LayoutGenerator
 import numpy as np
 import inspect
 
 def softmax(logits):
     e_x = np.exp(logits.T - np.max(logits))
     return (e_x / np.sum(e_x, axis=0)).T
 
-def get_base_env(mdp_params, env_params):
-    ae = AgentEvaluator(mdp_params=mdp_params, env_params=env_params)
-    return ae.env
 
-def get_mlp(mdp_params, env_params):
-    ae = AgentEvaluator(mdp_params=mdp_params, env_params=env_params)
-    return ae.mlp
+def get_base_ae(mdp_params, env_params, outer_shape=None, mdp_params_schedule_fn=None):
```

Comment:
if necessary you could keep the `get_base_env` and `get_mlp` functions (just have them call `get_base_ae` under the hood) but it seems you've already completely refactored them out so it probably wouldn't matter

Reply:
I added back `get_mlam` and `get_base_env` back in there :)

do a `ctr + shift + F` of `get_mlp` and replace it with `get_mlam`. Almost everything else should work